Assets/MixedRealityToolkit/InputModule/Prefabs/HoloLensCamera.prefab

```diff
@@ -52,7 +52,6 @@ GameObject:
   m_Component:
   - component: {fileID: 4000010560871102}
   - component: {fileID: 20000013304850028}
-  - component: {fileID: 92000010221946138}
```

Comment:
I thought we removed this prefab some time back?  How come it's back?

Reply:
Yup, this prefab was never removed.